### PR TITLE
vendor.xml: Add Turkish GSM operators

### PR DESCRIPTION
**Pull request by NukeMania**
configs extracted from poplar's carrierconfig.xml (officially last known phone in country that contains carrier configuration for turkish operators)

### CODE REVIEW

File: overlay/packages/apps/CarrierConfig/res/xml/vendor.xml [hunk 1]
```diff
@@ -623,23 +623,35 @@
     </carrier_config>
     <carrier_config mcc="286" mnc="01">
         <boolean name="carrier_volte_available_bool" value="true" />
-        <boolean name="carrier_volte_roaming_available_bool" value="false" />
+        <boolean name="carrier_wfc_ims_available_bool" value="true" />
+        <boolean name="carrier_default_wfc_ims_enabled_bool" value="true" />
         <int name="carrier_default_wfc_ims_mode_int" value="1" />
-        <int name="carrier_default_wfc_ims_roaming_mode_int" value="1" />
+        <int name="carrier_default_wfc_ims_roaming_mode_int" value="2" />
```
Comment (Haxk20) [inline overlay/packages/apps/CarrierConfig/res/xml/vendor.xml:629]: Why the change of value here ?

**Review by Haxk20 — APPROVED** [review]
Other then that one question about changing the roaming mode it looks OK.
Tho the commit message and PR title could be written better. For first contribution its OK.
EDIT: Thanks for clearing up the commit message.

File: overlay/packages/apps/CarrierConfig/res/xml/vendor.xml (hunk 1 above)
Comment (Haxk20) [inline overlay/packages/apps/CarrierConfig/res/xml/vendor.xml:629]: Why the change of mode here ?

**Review by NukeMania — COMMENTED** [review]
wrong path sorry

File: overlay/packages/apps/CarrierConfig/res/xml/vendor.xml (hunk 1 above)
Comment (NukeMania) [reply overlay/packages/apps/CarrierConfig/res/xml/vendor.xml:629]: value=2 indicates to wifi preferred in roaming mode rather than cellular mode
also stock's default value and turkish operators knows own people :) and it's more safe when someone using in roaming mode

Comment (Haxk20) [reply overlay/packages/apps/CarrierConfig/res/xml/vendor.xml:629]: Im aware of what it means. I was mostly asking why was the change of the mode changed but since its from stock we can assume its safe from the looks of it.

**Comment by Haxk20** [comment]
@kholk @Paulbouchara Click Click Clickity Click.
Also push it to branch ten :)
